CRM/Upgrade/Incremental/sql/5.43.alpha1.mysql.tpl

```diff
@@ -1 +1,3 @@
 {* file to handle db changes in 5.43.alpha1 during upgrade *}
+
+ALTER TABLE civicrm_contribution_recur MODIFY COLUMN modified_date DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT 'Last updated date for this record. mostly the last time a payment was received';
```

Comment:
@mattwire just wondering should this be timestamp not datetime? I note it currently is a datetime I think but for modified date should we change to timestamp here?

Reply:
It might be one of those ones where we made it datetime because of earlier mysql having limits around timestamp. I agreed with Timestamp being the right format